zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/FileSystemStorage.java

```diff
@@ -60,16 +59,25 @@
   public FileSystemStorage(ZeppelinConfiguration zConf, String path) throws IOException {
     this.zConf = zConf;
     this.hadoopConf = new Configuration();
+    URI zepConfigURI;
```

Comment:
Is the change in `FileSstemStorage.java` necessary ? IIUC, it is not necessary to do any change for this class.